lib/veri_report.ml

```diff
@@ -0,0 +1,56 @@
+open Core_kernel.Std
+open Bap.Std
+open Bap_traces.Std 
+
+module Map = String.Map
```

Comment:
Usually, it is a bad idea to hide generic `Set` and `Map` modules with a concrete module. Use better names. For example, for `Value.Set` instead of `Set` use `Values`. For `String.Map` here `Names` would work better.